tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
     docs, packaging, lint-py2, lint-py3, mypy,
-    py27, py34, py35, py36, py37, pypy
+    py{27,34,35,36,37,py,py3}-{functional-install,others}
```

Comment:
Can we still do tox -e py27 and tox -e py36?

If not, that'll be needed IMO.

Reply:
Good point, that needs updating. Do you know how to do that with tox?

Reply:
I'd just add `py{27,34,35,36,py,py3}` right above it.

It's a duplication I won't worry about since they're basically one under the other and a mismatch would be visible.

Reply:
Actually, I don't use tox much locally, but I think the commands you gave already work.

Reply:
Cool. :)